addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16541,3 +16541,13 @@ msgstr ""
 msgctxt "#38016"
 msgid "%d fps"
 msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38120"
+msgid "Support 8 channel DTS HD audio decoding"
+msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38121"
+msgid "This option supports 8 channel DTS HD decoding, but may use more CPU. It is only available when DTS and DTS-HD audio passthrough is disabled"
```

Comment:
It's not correct.

What about:
"Enables decoding of high quality DTS-HD audio streams. Note that this
increases CPU load and is only available when DTS and DTS-HD audio
passthrough are disabled."

@da-anda: Yes. It's currently one vs the other.

2015-05-10 17:35 GMT+02:00 da-anda <notifications@github.com>:

> In addons/resource.language.en_gb/resources/strings.po
> <https://github.com/xbmc/xbmc/pull/7102#discussion_r30003149>:
>
> > @@ -16541,3 +16541,13 @@ msgstr ""
> >  msgctxt "#38016"
> >  msgid "%d fps"
> >  msgstr ""
> > +
> > +#: system/settings/rbp.xml system/settings/imx6.xml
> > +msgctxt "#38120"
> > +msgid "Support 8 channel DTS HD audio decoding"
> > +msgstr ""
> > +
> > +#: system/settings/rbp.xml system/settings/imx6.xml
> > +msgctxt "#38121"
> > +msgid "This option supports 8 channel DTS HD decoding, but may use more CPU. It is only available when DTS and DTS-HD audio passthrough is disabled"
>
> @fritsch <https://github.com/fritsch> does this description sound good to
> you from a technical point or are there errors?
>
> Enables conversion of DTS-HD audio streams into a format supported by the connected audio device and will give bitperfect 8 channel audio fidelity on devices that support PCM. Note that this might increase CPU load and is only available when DTS and DTS-HD audio passthrough are disabled.
>
> What I wonder though is, why does DTS-HD decoding depend on DTS
> passthrough? Because DTS-HD does also contain a DTS core stream? What about
> AVRs that support DTS but not DTS-HD - can't I have DTS passthrough for
> videos with DTS track but decoding to PCM when I have a DTS-HD track?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/7102/files#r30003149>.
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B